data/data/bootstrap/baremetal/systemd/units/ironic.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Baremetal Deployment Ironic Services
+Wants=network-online.target crio.service
+After=network-online.target crio.service
+
+[Service]
+Type=exec
+ExecStart=/usr/local/bin/startironic.sh
+ExecStop=/usr/local/bin/stopironic.sh
+ConditionPathExists=!/etc/pivot/image-pullspec
```

Comment:
what's the significance of this condition? Why is this required?

Reply:
Good point, I copied the file from the previous coredns integration and assumed it was required to ensure the service only started after the ignition pivot is complete, but that may be incorrect.

@celebdor perhaps you can clarify, was this needed for the coredns service, or just a copy/paste addition we can safely remove?